### PR TITLE
Translation file for commands & tiny cleanup

### PR DESCRIPTION
**Pull request by Poseidon281**
I am working on translating all of the commands. If approved it can also be added to the Crowdin page

### CODE REVIEW

**Comment by lancepioch** [comment]
@Poseidon281 Thank you! Once this gets merged, CrowdIn will automatically pick it up within an hour.

**Comment by Poseidon281** [comment]
> @Poseidon281 Thank you! Once this gets merged, CrowdIn will automatically pick it up within an hour.

Only if the sync is enabled 2 way and not just the default sync only to the repo and not from. I dont know if we did that. Otherwise you have to manually sync new files from the repo

**Comment by Poseidon281** [comment]
Any way I still have to finish it so time enough

**Comment by Poseidon281** [comment]
> @Poseidon281 Thank you! Once this gets merged, CrowdIn will automatically pick it up within an hour.

The translation file for /var/www/pelican/app/Console/Commands is done. When merged you still need to manually sync the repo from Crowdin. That is not included in the normal sync standard i think.

**Comment by lancepioch** [comment]
I think pint issues are from me, I have to close and reopen the pr.

**Comment by lancepioch** [comment]
@Poseidon281 Awesome, thanks! I believe I've resolved the conflicts, however can you double check that it still works fine and run vendor/bin/pint if need be?

File: lang/en/commands.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    /* Not yet added to the panel
+    "debug" => [
+        "enable" => "Debug mode has been enabled",
+        "disable" => "Debug mode has been disabled",
+        "disable_option" => "Disable Debug mode",
+        "enable_option" => "Disable Debug mode",
+        "cancel_option" => "Cancel the Command",
+    ],
+    */
```
Comment (lancepioch) [inline lang/en/commands.php:13]:
```suggestion
```

File: lang/en/commands.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            'url' => 'The application URL MUST begin with https:// or http:// depending on if you are using SSL or not. If you do not include the scheme your emails and other content will link to the wrong location.',
+            'timezone' => "The timezone should match one of PHP\'s supported timezones. If you are unsure, please reference https://php.net/manual/en/timezones.php.",
+            //"lang" => "Choose a language you want to use on your panel.",
+            'settings_ui' => 'Enable UI based settings editor?',
+        ],
+        /*
+        "lang" => [
+        "question" => "What language do you want to use?",
+        ],
+        */
```
Comment (lancepioch) [inline lang/en/commands.php:16]:
```suggestion
        ],
```

File: lang/en/commands.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<?php
+
+return [
+    'appsettings' => [
+        'comment' => [
+            'author' => 'Provide the email address that eggs exported by this Panel should be from. This should be a valid email address.',
+            'url' => 'The application URL MUST begin with https:// or http:// depending on if you are using SSL or not. If you do not include the scheme your emails and other content will link to the wrong location.',
+            'timezone' => "The timezone should match one of PHP\'s supported timezones. If you are unsure, please reference https://php.net/manual/en/timezones.php.",
+            //"lang" => "Choose a language you want to use on your panel.",
```
Comment (lancepioch) [inline lang/en/commands.php:9]:
```suggestion
```

**Pull request merged**